oldlx/Trees/Engine.java

```diff
@@ -76,7 +76,7 @@
   final ChannelShrubLevels[] channelShrubLevels = new ChannelShrubLevels[Engine.NUM_TOTAL_CHANNELS];
   final BasicParameter dissolveTime = new BasicParameter("DSLV", 400, 50, 1000);
   final BasicParameter drumpadVelocity = new BasicParameter("DVEL", 1);
-  
+
```

Comment:
my text editor trims whitespace on save -- i can turn this off if you don't want me polluting the git history, just lmk